depletion/casl_chain.py

```diff
@@ -278,3 +275,9 @@
     'Dy165': (False, 3, 2, None),
     'Ho165': (True, 3, 3, [('Dy165_m1', 0.022, 2), ('Ho165', 1.000, 1)])
 }
+
+# When building the depletion chain, we generally ensure that the sum of
+# branching ratios is one. However, in a few cases, this doesn't make sense and
+# would end up producing incorrect concentrations downstream by overproducing
+# certain nuclides.
+DONT_MODIFY_BR = {'Rh102', 'Ag110', 'I128', 'Tm170', 'Am244_m1'}
```

Comment:
Following from my comment on the comment, I would recommend
```suggestion
UNMODIFIED_DECAY_BR = {'Rh102', 'Ag110', 'I128', 'Tm170', 'Am244_m1'}
```
just to add a little more explicitness.